gatsby/src/components/DocsLayout/index.js

```diff
@@ -14,7 +14,9 @@ const DocsLayout = ({ data: { mdx } }) => (
     </Box>
 
     <Box gridArea="main" as="main">
-      <Heading level={1} lined>{mdx.frontmatter.title}</Heading>
+      <Heading level={1} lined>
```

Comment:
FYI: I think this prop would be easier to understand if we would name it to underlined

Reply:
that might conflict with text-decoration. but yeah i get you.

basically we're noticing the stuff stas did for the website won't work for the docs. have to change values n such